src/main/java/net/minecraftforge/network/HandshakeHandler.java

```diff
@@ -99,6 +105,8 @@ static boolean tickLogin(Connection networkManager)
     private Map<ResourceLocation, ForgeRegistry.Snapshot> registrySnapshots;
     private Set<ResourceLocation> registriesToReceive;
     private Map<ResourceLocation, String> registryHashes;
+    private boolean negotiationStarted;
```

Comment:
It seems like this field is never initialized.

Reply:
The field is implicitly false, however I can explicitly set it if you'd like?

Reply:
That would be much better for reading at a glance, yeah.